modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -722,6 +722,70 @@ definition {
 		}
 	}
 
+	@description = "This test asserts an account member can search accounts they belong to"
+	@priority = "3"
+	test AccountMemberCanSearchForAnAccount {
+		task ("Given a new account is created and a new account user is assigned to two accounts") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Account Test",
+				accountEntryType = "Business");
+
+			JSONAccountEntryUser.addAccountEntryUser(
+				accountEntryName = "Account Test",
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			Navigator.gotoPage(pageName = "2B Account Page");
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Accounts.assignUsers(
+				accountName = "Account Name",
+				assigneeName = "user1");
+		}
+
+		task ("When the new account member signs in and views the accounts they are a part of through the Account Management widget") {
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "user1@liferay.com");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "user1@liferay.com",
+				userLoginFullName = "user1 user1");
+
+			Navigator.gotoPage(pageName = "2B Account Page");
+
+			Search.searchPG(
+				portletName = "Account Management",
+				searchTerm = "Test");
+		}
+
+		task ("Then the member can use the search bar to filter out the accounts they belong to") {
+			Search.searchPG(
+				portletName = "Account Management",
+				searchTerm = "Test");
+
+			LexiconEntry.viewEntryName(rowEntry = "Account Test");
+
+			LexiconEntry.viewNoEntry(rowEntry = "Account Name");
+
+			Search.searchPG(
+				portletName = "Account Management",
+				searchTerm = "Name");
+
+			LexiconEntry.viewEntryName(rowEntry = "Account Name");
+
+			LexiconEntry.viewNoEntry(rowEntry = "Account Test");
+		}
```

Comment:
Would it be better to add a new account with no user, and confirm the account not shows up when the use login? Thanks.

Reply:
I think that's covered in 2BAccount#CanAccessAListOfAccountsTheUserBelongsTo. For this test, I think just asserting the search bar works for accounts might be enough? I am pushing up the changes for your other comments, but will wait for your final decision on this. Thank you!

Reply:
Cool, thanks for double checking it Eric, you're right, that would be duplicate to the other test.